account_invoice_refund_link/views/account_invoice_view.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2004-2011 Pexego Sistemas Informáticos. (http://pexego.es)
+     Copyright 2014 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+     Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+<record id="view_customer_account_invoice_add_refunds_details_form" model="ir.ui.view">
+    <field name="name">Account invoice (customer) | add Refunds details (form)</field>
+    <field name="model">account.move</field>
+    <field name="inherit_id" ref="account.view_move_form"/>
+    <field name="arch" type="xml">
+        <notebook position="inside">
+            <page name="refunds"
+                  string="Refunds"
+                  attrs="{'invisible':[('type', 'not in', ['in_invoice', 'out_invoice'])]}"
+            >
+                <field name="refund_invoice_ids" nolabel="1">
+                    <tree decoration-info="state == 'draft'"
+                          decoration-muted="state == 'cancel'">
+                        <field name="partner_id" invisible="1"/>
+                        <field name="type" invisible="1"/>
+                        <field name="invoice_date"/>
+                        <field name="name"/>
+                         <field name="invoice_origin"/>
+                        <field name="ref"/>
+                        <field name="amount_total" string="Total"/>
+                        <field name="state" string="Status"/>
+                    </tree>
+                </field>
+            </page>
+        </notebook>
+        <field name="ref" position="after">
+            <field name="type" invisible="1"/>
+            <field name="reversed_entry_id"
+                string="Invoice reference"
+                readonly="1"
+                attrs="{'invisible': [
+                    '|',
+                    ('reversed_entry_id', '=', False),
+                    ('type', 'not in', ('in_refund', 'out_refund'))
```

Comment:
I think this part is not needed as the tab is not shown in that case, but you should use a better xpath for being sure the element is placed inside the page that belongs to invoices.

Reply:
The field I pretend to locate is the one that shows in a Refund, the name of the original Invoice and the reason for the Refund. It is part of the General Information of the Refund form view, after that, I thought the link to the original invoice could be located.
![Screenshot from 2020-02-03 20-21-15](https://user-images.githubusercontent.com/38267832/73711038-a2b47280-46d3-11ea-9305-7f4fb23324ad.png)